components/Dialogs/examples/DialogsThemedPresentationExampleViewController.swift

```diff
@@ -0,0 +1,202 @@
+                         toItem: titleLabel,
+                         attribute: .bottom,
+                         multiplier: 1.0,
+                         constant: 10.0),
+      NSLayoutConstraint(item: bodyLabel,
+                         attribute: .left,
+                         relatedBy: .equal,
+                         toItem: self.view,
+                         attribute: .left,
+                         multiplier: 1.0,
+                         constant: 5.0),
+      NSLayoutConstraint(item: bodyLabel,
+                         attribute: .right,
+                         relatedBy: .equal,
+                         toItem: self.view,
+                         attribute: .right,
+                         multiplier: 1.0,
+                         constant: -5.0),
+      NSLayoutConstraint(item: confirmButton,
+                         attribute: .top,
+                         relatedBy: .equal,
+                         toItem: bodyLabel,
+                         attribute: .bottom,
+                         multiplier: 1.0,
+                         constant: 10.0),
+      NSLayoutConstraint(item: confirmButton,
+                         attribute: .bottom,
+                         relatedBy: .lessThanOrEqual,
+                         toItem: self.view,
+                         attribute: .bottom,
+                         multiplier: 1.0,
+                         constant: 0.0),
+      NSLayoutConstraint(item: confirmButton,
+                         attribute: .centerX,
+                         relatedBy: .equal,
+                         toItem: self.view,
+                         attribute: .centerX,
+                         multiplier: 1.0,
+                         constant: 0.0),
+      ])
+  }
+
+  override var preferredContentSize: CGSize {
+    get {
+      return CGSize(width:200.0, height:150.0);
+    }
+    set {
+      super.preferredContentSize = newValue
+    }
+  }
+
+}
+
+class DialogsThemedPresentationExampleViewController: UIViewController {
+
+  private let materialButton = MDCButton()
+  private let transitionController = MDCDialogTransitionController()
+  var containerScheme = MDCContainerScheme()
+
+  override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
+    super.init(nibName: nibNameOrNil, bundle: nibBundleOrNil)
+    containerScheme.colorScheme = MDCSemanticColorScheme(defaults: .material201804);
+  }
+
+  required init?(coder aDecoder: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+
+    view.backgroundColor = containerScheme.colorScheme?.backgroundColor ??
+      MDCSemanticColorScheme(defaults: .material201804).backgroundColor
+
+    materialButton.translatesAutoresizingMaskIntoConstraints = false
+    materialButton.setTitle("Custom Alert With Themed Presentation", for: .normal)
+    materialButton.setTitleColor(UIColor(white: 0.1, alpha:1), for: .normal)
```

Comment:
Please use the Buttons Theming Extension to theme.

Reply:
Addressed in 7c3b8a2b897c3552b19b33a3ae50a2c7d9c9ee77